manifests/kustomize/README.md

```diff
@@ -29,15 +29,12 @@ To get latest kubectl, visit [here](https://kubernetes.io/docs/tasks/tools/insta
 
 ## Change deploy namespace
 To deploy Kubeflow Pipelines in namespace FOO,
-- Edit [kustomization.yaml](namespaced-install/kustomization.yaml) namespace section to FOO
+- Edit [kustomization.yaml](env/dev/kustomization.yaml) namespace section to FOO
```

Comment:
This should now mention editing **manifests/kustomize/env/gcp/ml-pipeline-apiserver-deployment-patch.yaml** as well since it requires a user to provide their bucket name.

Reply:
thanks. will do it in a separate PR. I'm still figuring out the best way to configure gcp deployment. The goal is to have a single file for user to edit their configuration. 
(currently spreading in multiple files)